test/convenience-method-test.js

```diff
@@ -212,6 +212,362 @@ module.exports = {
           }, 1000);
         }*/
   },
+  ".rescheduleJob(job, {...})": {
+    "Reschedule jobs from object based to object based": function(test) {
+      test.expect(3);
+
+      var job = new schedule.scheduleJob({
+        second: null
+      }, function() {
+        test.ok(true);
+      });
+
+      setTimeout(function() {
+        schedule.rescheduleJob(job, {
+          minute: null
+        });
+      }, 3250);
+
+      setTimeout(function() {
+        job.cancel();
+        test.done();
+      }, 5000);
+
+      clock.tick(5000);
```

Comment:
All these tests are great. It would be a nice addition to be able to check the # of times the job triggers before and after the rescheduling is performed